client/client.go

```diff
@@ -278,9 +278,7 @@ func (c *Client) processChannelRequests() {
 				responsePayload = new(agentpb.StartActionResponse)
 
 			case managementpb.ActionType_MYSQL_EXPLAIN:
-				pp := p.GetMysqlExplainParams()
-				format := actions.MysqlExplainOutputFormat(pp.OutputFormat)
```

Comment:
Here we implicitly depended on `actions.MysqlExplainOutputFormat` and `agentpb.MysqlExplainOutputFormat` to have the same values. Change in one of them will not be noticed by compiler or unit tests but will break this code. That was too fragile.